src/commands/org/create/sandbox.ts

```diff
@@ -175,6 +174,14 @@ export default class CreateSandbox extends SandboxCommandBase<SandboxCommandResp
       delete sandboxReq.ActivationUserGroupName;
     }
 
+    // Get source sandbox features if cloning
+    if (srcSandboxName ?? srcId) {
+      const sourceFeatures = await this.flags['target-org'].getSandboxFeatures(srcSandboxName ?? srcId);
```

Comment:
I would move the `getSandboxFeatures` function that is in sfdx-core to this file as a helper.  It would first call `querySandboxInfo()` to look for features.  If found on SandboxInfo then we'd stop and use it.  If not, we'd then `querySandboxProcess()` for features.  If found use it and if not there are no features.